### ISSUE TITLE
Find a way not to use <style> in bFormFile's template

### ISSUE THREAD

**Issue by gpyh** [bootstrap-vue/bootstrap-vue#175]
See title.

This is webpack's error message (or should I say vue-loader's):

```
ERROR in ./~/vue-loader/lib/template-compiler.js?{"id":"data-v-d1e1f006"}!./~/vue-loader/lib/selector.js?type=template&index=0!../bootstrap-vue/lib/components/form-file.vue

  Error compiling template:

  <label :lang="lang" :class="['custom-file',unique_class]">

      <!-- Inject strings -->
      <style>
          .{{unique_class}} .custom-file-control::after {
              content: "{{selectedLabel}}" !important;
          }

          .{{unique_class}} .custom-file-control::before {
              content: "{{chooseLabel}}" !important;
          }
      </style>

      <input type="file" :id="id" class="custom-file-input" @change="onFileChange">
      <span class="custom-file-control"></span>

  </label>

  - Templates should only be responsible for mapping the state to the UI. Avoid placing tags with side-effects in your templates, such as <style>, as they will not be parsed.
```

**Comment by pi0** [comment]
Styles are fixed as well :) (See #174)

**Comment by gpyh** [comment]
Fixed indeed. Thanks.
